.circleci/config.yml

```diff
@@ -20,14 +20,22 @@ jobs:
     steps:
       - checkout
       - run: .circleci/clone_tracker.sh
-      - run: make integration-tests debug=true
+      - run: |
+          ulimit -c unlimited
```

Comment:
For both of the `ulimit -c` commands that are added in this YAML file, is the value of the sysctl `kernel.core_pattern` variable something useful?

Reply:
Already merged, but for posterity: https://discuss.circleci.com/t/core-dumps-now-supported-on-circleci-2-0/13713

> With the limit removed, anytime a program crashes a dump file will be created in the current working directory. The name it will be given, as specified in /proc/sys/kernel/core_pattern will be core.%p.%E where %p is the process id and %E is the pathname of the executable.